### ISSUE TITLE
Seg fault when using RecStates w/ UTA

### ISSUE THREAD

**Issue by DrRyanC** [flexible-atomic-code/fac#685]
Hello. 

I receive a segmentation fault when using the RecStates function with the SetUTA for configuration-averaged recombined states. RecStates works fine in detailed level accounting when UTA isn't active, but faults with UTA. I have tried multiple variations of SetRecPWOptions, SetRecSpectator, and PrepAngular calls to mitigate the memory pressure of the RecState, but with no success. 

Is it possible to use these functions together? Since SetUTA disables the angular momentum coupling, does the RecStates & PrepAngular become incompatible since they explicitly use the angular coefficients between states? The RecStates function is desired for DR calculations using a configuration averaged UTA level accounting. I am using FAC 1.1.5 (master branch). Here is a relevant portion of the script being used for a simple H-like Ar calculation:

el = str(sys.argv[1])
q = int(sys.argv[2])

fac.SetAtom(el)
fac.SetUTA(1)
pref = "%s%02d"%(el, q)

fac.Config('g0', '1s1')
fac.Config('exc2', '2*1')
fac.Config('exc3', '3*1')

fac.ConfigEnergy(0)
fac.OptimizeRadial(['g0'])
fac.ConfigEnergy(1)
fac.Structure(pref+'b.en', ['g0','exc2','exc3'])

fac.PrepAngular(['g0'])
fac.PrepAngular(['exc2'])
fac.PrepAngular(['exc3'])
fac.PrepAngular(['g0'], ['exc2', 'exc3'])

fac.SetRecPWOptions(8)
fac.SetRecSpectator(4, 10)
for n in range(4, 10):
    fac.RecStates(pref+'b.en', ['g0','exc2','exc3'], n)

fac.MemENTable(pref+'b.en')
fac.PrintTable(pref+'b.en', pref+'RecLev.en')

Thank you!

**Comment by mfgu** [comment]
indeed, RecStates wasn't designed for UTA states. I made some changes in the master branch to make it work. computationally, using RecStates would be identical to specifying the configurations directly when in UTA.

**Comment by DrRyanC** [comment]
Thank you for the clarification and for taking the time to revise this capability!
